docs/src/content/docs/fr/guides/sidebar.mdx

```diff
@@ -0,0 +1,488 @@
+	]}
+/>
+
+### Groupes
+
+Vous pouvez donner de la structure à votre barre latérale en regroupant des liens connexes sous un en-tête rétractable.
+Les groupes peuvent contenir à la fois des liens et d'autres sous-groupes.
+
+Ajoutez un groupe en utilisant un objet avec les propriétés `label` et `items`.
+Le `label` sera utilisé comme en-tête pour le groupe.
+Ajoutez des liens ou des sous-groupes au tableau `items`.
+
+```js
+starlight({
+	sidebar: [
+		// Un groupe de liens avec le label "Guides".
+		{
+			label: 'Guides',
+			items: [
+				{ label: 'Composants', link: '/fr/guides/components/' },
+				{ label: 'Internationalisation (i18n)', link: '/fr/guides/i18n/' },
+				// Un groupe de liens imbriqué.
+				{
+					label: 'Mise en forme',
+					items: [
+						{ label: 'CSS', link: '/fr/guides/css-and-tailwind/' },
+						{ label: 'Tailwind', link: '/fr/guides/css-and-tailwind/' },
+						{ label: 'Shiki', link: '/fr/guides/css-and-tailwind/' },
+					],
+				},
+			],
+		},
+	],
+});
+```
+
+La configuration ci-dessus génère la barre latérale suivante :
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Guides',
+			items: [
+				{ label: 'Composants', link: '/fr/guides/components/' },
+				{ label: 'Internationalisation (i18n)', link: '/fr/guides/i18n/' },
+				{
+					label: 'Mise en forme',
+					items: [
+						{ label: 'CSS', link: '/fr/guides/css-and-tailwind/' },
+						{ label: 'Tailwind', link: '/fr/guides/css-and-tailwind/' },
+						{ label: 'Shiki', link: '/fr/guides/css-and-tailwind/' },
+					],
+				},
+			],
+		},
+	]}
+/>
+
+### Groupes générés automatiquement
+
+Starlight peut générer automatiquement un groupe dans votre barre latérale en fonction d'un répertoire de votre documentation.
+Cela est utile lorsque vous ne souhaitez pas entrer manuellement chaque élément de la barre latérale dans un groupe.
+Les pages seront triées par ordre alphabétique en fonction du nom de fichier par défaut.
+
+Ajoutez un groupe généré automatiquement en utilisant un objet avec les propriétés `label` et `autogenerate`. La configuration de `autogenerate` doit spécifier le répertoire à utiliser pour les entrées de la barre latérale avec la propriété `directory`. Par exemple, avec la configuration suivante :
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Guides',
+			// Génère automatiquement un groupe de liens pour le répertoire "guides".
+			autogenerate: { directory: 'guides' },
+		},
+	],
+});
+```
+
+Et la structure de fichiers suivante :
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+        - advanced/
+          - project-structure.md
+
+</FileTree>
+
+La barre latérale suivante sera générée :
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Guides',
+			items: [
+				{ label: 'Composants', link: '/guides/components/' },
+				{ label: 'Internationalisation (i18n)', link: '/guides/i18n/' },
+				{
+					label: 'advanced',
```

Comment:
This label should probably be translated as well here, shouldn't it?

```suggestion
					label: 'Avancé',
```

Reply:
> This label should probably be translated as well here to avoid the following rendering, shouldn't it?

It should not in this case. The entire "Guides" group is auto generated meaning pages and groups are based on the file structure:

- Pages like `components.md`, `i18n.md`, `project-structure.md` have their label picked from the page frontmatter titles or `sidebar.label` property meaning it can be translated in the page.
- Subdirectories like `advanced/` are displayed as-is meaning that to get "Avancé" the directory on disk should actually be `Avancé/`.

Maybe directory names could be translated but it feels a bit weird to me considering that some won't be translated like `src`, `content`, etc. and even more with an `é` in the directory name.

Reply:
> Subdirectories like advanced/ are displayed as-is meaning that to get "Avancé" the directory on disk should actually be Avancé/.

Oh OK, I understand now the subtlety here. Found it so weird while reading this page. Thanks for the clarification.